src/components/notifications/AppSelector/AppSelector.scss

```diff
@@ -161,12 +166,12 @@
       padding: 1rem 1rem 1rem 1rem;
       border-radius: 1em;
       position: relative;
-      transition: all 250ms ease-in-out;
+      transition: background-color 0.1s linear;
```

Comment:
While we set the transition property, we should always set which properties will be animated, instead using `all`. 

Also, it's always good for performance practice to use `will-change`. It's a property that let the browser know which props will be animated. And the browser allocates itself depending on this and will help to make the animations more performant.

- https://web.dev/articles/animations-guide

Reply:
Awesome, always love improving performance